kdbg/pgmsettings.cpp

```diff
@@ -16,9 +17,13 @@
 
 
 ChooseDriver::ChooseDriver(QWidget* parent) :
-	QWidget(parent)
+	QWidget(parent),
+	m_disassLabel( new QLabel(i18n("Disassembly flavor:"), this)),
+	m_disassComboBox(new QComboBox(this)),
+	m_isX86(false)
```

Comment:
Please initialize this member in the class definition (in the header file), not here. It more modern C++:
```
bool m_isX86 = false;
```

Concerning `m_disassLabel` and `m_disassComboBox`, please follow the precedent set by `m_debuggerCmd` and initialize them in the body below. That would mean that `m_disassLabel` would not have to be a member variable.